source/features/show-github-action-next-time.tsx

```diff
@@ -0,0 +1,71 @@
+import './show-github-action-next-time.css';
+import React from 'dom-chef';
+import cache from 'webext-storage-cache';
+import select from 'select-dom';
+import elementReady from 'element-ready';
+import * as pageDetect from 'github-url-detection';
+import {parseCron} from '@cheap-glitch/mi-cron';
+
+import features from '.';
+import * as api from '../github-helpers/api';
+import {getRepoURL, getRepoGQL, getCurrentBranch} from '../github-helpers';
+
+const getWorkflows = cache.function(async (): Promise<Array<{[index: string]: {[index: string]: string}}>> => {
+	const {repository: {object: {entries: workflows}}} = await api.v4(
+		`repository(${getRepoGQL()}) {
+			object(expression: "${getCurrentBranch()}:.github/workflows") {
+				... on Tree {
+					entries {
+						object { ... on Blob { text } }
+					}
+				}
+			}
+		}
+	`);
+
+	return workflows;
```

Comment:
Always parse the API responses before returning them so you don’t have to type the entire API (perhaps giving false expectations). You can inline parseYamls in here to do that.